packages/plugins/graphql/server/src/bootstrap.ts

```diff
@@ -25,7 +30,7 @@ const merge = mergeWith((a, b) => {
 const useUploadMiddleware = (strapi: Strapi, path: string): void => {
   const uploadMiddleware = graphqlUploadKoa();
 
-  strapi.server.app.use((ctx, next) => {
+  strapi.server.app.use((ctx: Context, next: Next) => {
```

Comment:
Again, it's weird that you have to specify this, app.use should already be typed :(

Reply:
Yeah, I just updated from v5/main and now it's not needed anymore. Must have been something weird going on there.